### ISSUE TITLE
Format problem with rendering script

### ISSUE THREAD

**Issue by IntegralTeam** [One-2-3-45/One-2-3-45#50]
Hi! I have a plan to train model on custom dataset, and i faced a format problem:
- Objaverse LVIS (100Gb) has structure like this:
000-***/hash-model-id/{model_render_images}

I have about 50 models of .glb format. I've managed to render it with launch_render_eval.py to get 24 png's per model, but found out it's not appropriate format for training. Where can i find script, which could render 8 views x 5 angles x 3 images (orig, depth, depth_mm) per image?

in other words, what's the full training pipeline on custom dataset?

**Comment by IntegralTeam** [comment]
https://github.com/One-2-3-45/One-2-3-45/issues/24
Uncommenting [this](https://github.com/One-2-3-45/One-2-3-45/blob/ee278d0964ba2d3bf66f94d05da0f83a747ba6c1/render/single_render_eval.py#L204) line of code also doesn't help

**Comment by Dustinpro** [comment]
That is a rendering script for evaluation. You may modify its polar_angles and azimuths to render the images needed for training.
